app/src/androidTest/java/com/example/concordia_campus_guide/activities/CalendarNotificationTest.java

```diff
@@ -1,4 +1,4 @@
-package com.example.concordia_campus_guide.activities;
+package com.example.concordia_campus_guide.Activities;
```

Comment:
I think this is supposed to be small letter. Could you confirm @tarekait1996

Reply:
It is causing the tests to not be recognized so nothing is working @narrapangan @viveanban @tarekait1996

Reply:
Ok, for some reason I had a path with capital `Activities` but now its `activities`, so its working now